tests/k6/tests/enduser/dialogSystemLabelLog.js

```
@@ -0,0 +1,60 @@
+import {
+    describe, expect, expectStatusFor, getEU, putEU, postSO, putSO, purgeSO
+} from '../../common/testimports.js'
+
+import {default as dialogToInsert} from '../serviceowner/testdata/01-create-dialog.js';
+
+export default function () {
+    let dialogId;
+    let dialog;
+    describe('Arrange: Create a dialog to test against', () => {
+        let d = dialogToInsert();
```

Biome

[error] This let declares a variable that is only assigned once.

'd' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)